lib/tpl.js

```diff
@@ -57,6 +57,12 @@ async function getTpl(tplPath) {
   return tpl;
 }
 
+function validateYmlName(tplPath) {
+  if (!(path.basename(tplPath).endsWith('.yml') || path.basename(tplPath).endsWith('.yaml'))) {
+    throw new Error(red(`The -t argument:${tplPath} must end in '.[yml|yaml]'.`));
```

Comment:
The template file name must end with yml or yaml

Reply:
done